src/char/sorceress/sorceress.py

```diff
@@ -32,6 +33,40 @@ def pick_up_item(self, pos: Tuple[float, float], item_name: str = None, prev_cas
             return cast_start
         else:
             return super().pick_up_item(pos, item_name, prev_cast_start)
+            
+    def select_by_template(
```

Comment:
Is there a fail save in case telekensis is out of range? We will just try tele forever / until timeout now, right?

Reply:
I havent seen any cases out of range